app/upgrades/tv3/upgrades.go

```diff
@@ -0,0 +1,88 @@
+package tv3
```

Comment:
nit, we can move this to `app/upgrades/testnet/v*/upgrades.go` and the mainnet ones to `app/upgrades/mainnet/v*/upgrades.go`

Reply:
the package name can remain the same